src/main/java/org/bitsofinfo/hazelcast/discovery/docker/swarm/DockerSwarmDiscoveryStrategy.java

```diff
@@ -28,11 +28,10 @@
     /**
      * Constructor
      *
-     * @param localDiscoveryNode
      * @param logger
      * @param properties
      */
-    public DockerSwarmDiscoveryStrategy(DiscoveryNode localDiscoveryNode, ILogger logger, Map<String, Comparable> properties) {
```

Comment:
Do not alter any public signatures/constructors. This could break others using this library in use-cases specific to them such as programmatically configuring the stack.

Reply:
The passed DiscoveryNode is never used.  Surely best to clean that up with an appropriate release note?  Happy to revert if you wish....

Reply:
Please remove it